src/pages/filter/dtree/components/query-builder/ui/next-step/content-code/content-code.module.css

```diff
@@ -0,0 +1,36 @@
+.content-code {
+  display: flex;
+  height: auto;
+  width: 33.333%;
+  margin-left: 8px;
+
+  &_wrapper {
+    width: 100%;
+    height: auto;
+    padding: theme('spacing.2');
+    background: theme('colors.blue.secondary');
+    color: theme('colors.white');
+    font-size: theme('spacing.3');
```

Comment:
It's not good to use `spacing` for `font-size`. You can `@apply typography-...` component or create your own in `components.js`